test/functional/news_letter_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# TODO: We had some functional tests that tested CRUD (create-update-delete)
+# functionality of the teacher sets controller.  They were deleted in
```

Comment:
Let's be a little bit more clear.  Some quesitons I have are: 
Who are "they"?  
what are the changes that reflect this new version?

Reply:
@daryanypl Sorry that is copy paste comments. I will remove it.